pkg/panels/perses/perses.go

```diff
@@ -95,6 +102,13 @@ func HTTPRequestsRatePanel(datasourceName string, labelMatchers ...promql.LabelM
 				Position: timeSeriesPanel.RightPosition,
 				Mode:     timeSeriesPanel.TableMode,
 			}),
+			timeSeriesPanel.WithVisual(timeSeriesPanel.Visual{
+				Display:      timeSeriesPanel.LineDisplay,
+				ConnectNulls: false,
+				LineWidth:    0.25,
+				AreaOpacity:  0.5,
```

Comment:
Maybe the request and error rates should be stacked? 
Helps view the 2xx/5xx response rates in relation to each other

Reply:
Ah not quite, sorry for the confusion, I meant,

One HTTP Request Rate panel that shows all response codes but stacked, using `Stack:  timeSeriesPanel.AllStack,`

And one HTTP Error Rate panel that shows rate (or percentage) of 5xx codes (but also stacked visually so you can see types of 5xx)

This is kind of what we follow for Thanos dashboards. For eg. instant query https://github.com/perses/community-dashboards/blob/main/pkg/panels/thanos/thanos_query.go#L15 and instant query error ratio https://github.com/perses/community-dashboards/blob/main/pkg/panels/thanos/thanos_query.go#L50

Both are stack, one highlights all responses, other only errors, but you can see the proportion of response codes next to each other as they are visually stacked.

Reply:
thanks a lot for the explanation and pointer. fixed in 592396073f62a69d7ca2549395369b71f68c0283 :)